<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeakDetectionDialog</class>
 <widget class="QDialog" name="PeakDetectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>773</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Peak Detection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="3" column="1" alignment="Qt::AlignLeft">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Peak Detection</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="1">
        <widget class="QGroupBox" name="featureOptions">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Feature Detection</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="4" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Time Domain Resolution (scans)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="ppmStep">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>20.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="featureMustHaveMs2">
            <property name="text">
             <string>Must Have MS2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Mass Domain Resolution (ppm)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="rtStep">
            <property name="statusTip">
             <string>mz-rt matrix binning</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string> scans</string>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>5.000000000000000</double>
            </property>
            <property name="value">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="excludeIsotopicPeaks">
            <property name="text">
             <string>Excluded Istopic Peaks</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QGroupBox" name="eicOptions">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>EIC Processing</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>EIC Smoothing</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Peak Grouping (Max Group Rt Difference)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="grouping_maxRtDiff">
            <property name="suffix">
             <string> min</string>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="eic_smoothingWindow">
            <property name="suffix">
             <string> scans</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Baseline Calculation</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="1" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Baseline Smooting </string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="baseline_smoothing">
               <property name="suffix">
                <string> scans</string>
               </property>
               <property name="maximum">
                <number>1000000</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="baseline_quantile">
               <property name="suffix">
                <string> %</string>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>80</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Drop top x%  intensities from chromatogram</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="dbOptions">
         <property name="title">
          <string>Compound Database</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QCheckBox" name="matchRt">
            <property name="text">
             <string>Match Retention Times</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="reportIsotopes">
            <property name="text">
             <string>Report Isotopic Peaks</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="compoundDatabase"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>EIC Extraction Window  +/- PPM </string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QDoubleSpinBox" name="compoundPPMWindow">
            <property name="suffix">
             <string> ppm</string>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="value">
             <double>20.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QSpinBox" name="eicMaxGroups">
            <property name="suffix">
             <string> best</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QDoubleSpinBox" name="compoundRTWindow">
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string> min</string>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Compound Retention Time Matching Window</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Number of Reported Groups per Compound</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Compound Subset</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="compoundMustHaveMS2">
            <property name="text">
             <string>Must Have MS2</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QCheckBox" name="reportAdducts">
            <property name="text">
             <string>Report Adducts</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Peak Scoring</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Peak Scoring</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Min. Signal/BaseLine Ratio</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QDoubleSpinBox" name="sigBlankRatio">
            <property name="maximum">
             <double>10000000.000000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="label_100">
            <property name="text">
             <string>Peak Classifier Model File</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Min. Signal/Blank Ratio</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Min. Good Peak / Group</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="minGoodGroupCount">
            <property name="suffix">
             <string> peaks</string>
            </property>
            <property name="maximum">
             <number>100000000</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="minNoNoiseObs">
            <property name="suffix">
             <string> scans</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000000</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Min. Peak Width</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QLineEdit" name="classificationModelFilename">
            <property name="text">
             <string>default.model</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Min. Peak Intensity</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QDoubleSpinBox" name="minGroupIntensity">
            <property name="suffix">
             <string> ions</string>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="value">
             <double>5000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="sigBaselineRatio">
            <property name="maximum">
             <number>10000000</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QPushButton" name="loadModelButton">
            <property name="text">
             <string>Load Model</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Fragmentation Matching</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="1" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Fragment PPM Tolr</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="fragMinScore">
            <property name="maximum">
             <double>10000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Match at Least X  Peaks</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="fragMinPeaks">
            <property name="suffix">
             <string/>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Minumum Score</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="fragScoringAlgorithm"/>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="productPpmTolr">
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string> ppm</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>20.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Scoring Algorithm</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>productPpmTolr</zorder>
         <zorder>label_17</zorder>
         <zorder>label_19</zorder>
         <zorder>label_18</zorder>
         <zorder>fragScoringAlgorithm</zorder>
         <zorder>fragMinScore</zorder>
         <zorder>label_20</zorder>
         <zorder>fragMinPeaks</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Output Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outputDirName"/>
        </item>
        <item>
         <widget class="QPushButton" name="setOutputDirButton">
          <property name="text">
           <string>Set Directory</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="2">
          <widget class="QPushButton" name="computeButton">
           <property name="text">
            <string>Find Peaks</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="statusText">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
